### PR TITLE
[MM-27371] Bump mattermost-redux version

### PR DESCRIPTION
**Pull request by ethervoid**
#### Summary

Update to the last redux version to include changes that fixes MM-27371

#### Related Pull Requests

- Has server changes (please link here)
- Has redux changes (please link here)
- Has mobile changes (please link here)

#### Screenshots

https://mattermost.atlassian.net/browse/MM-27371

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
     "localforage-observable": "2.0.1",
     "mark.js": "8.11.1",
     "marked": "github:mattermost/marked#87769262aa02e1784570f61f4f962050e07cc335",
-    "mattermost-redux": "github:mattermost/mattermost-redux#7522d68c6a61a5bf535672670ef6195e610f2fa0",
+    "mattermost-redux": "github:mattermost/mattermost-redux#40fd505dcc0783c5af479642beaa566fbe5515ce",
```
Comment (saturninoabril) [inline package.json:40]: @ethervoid It should have different commit for master and release-5.26.
- master should be https://github.com/mattermost/mattermost-redux/commit/3f5b8566208b7b3116fb20b4ad8278ddbae2a0a3
- release-5.26 should be https://github.com/mattermost/mattermost-redux/commit/40fd505dcc0783c5af479642beaa566fbe5515ce

**Comment by ethervoid** [comment]
@amyblais removed the cherry-pick because I had to create a different branch for 5.26 pointing to another commit https://github.com/mattermost/mattermost-webapp/pull/6036

**Comment by sudheerDev** [comment]
@ethervoid Hey, The PR is not tested by QA.

If I am raising a PR for redux changes then i make a redux PR an a web PR with the redux pr commit to have QA test on the spinmint before merging the redux PR. This way the change is tested by QA before it gets into redux master. 

Another way to do it is to wait for webpr if not redux pr to have it tested.

**Comment by ethervoid** [comment]
@sudheerDev hi! This PR and the redux one was tested and approved by @saturninoabril on behalf of QA.

**Comment by saturninoabril** [comment]
Yes, I tested the PR locally per changes at https://github.com/mattermost/mattermost-redux/pull/1209

**Comment by sudheerDev** [comment]
> @sudheerDev hi! This PR and the redux one was tested and approved by @saturninoabril on behalf of QA.

Aah got it. Thanks

> So if I’ve understood correctly the way to do redux-webapp is to create both PRs then test first the webapp one and when that works then merge redux and then webapp?

Yup, that is how most devs do. There is no concrete process though.

**Pull request merged**